tests/Shared/WorkloadTesting/BuildEnvironment.cs

```diff
@@ -54,16 +54,17 @@ public class BuildEnvironment
     public static BuildEnvironment ForCurrentSdkOnly => s_instance_90.Value;
     public static BuildEnvironment ForCurrentSdkAndPreviousRuntime => s_instance_90_80.Value;
 
-    public static BuildEnvironment ForDefaultFramework { get; } = DefaultTargetFramework switch
-    {
-        TestTargetFramework.Previous => ForPreviousSdkOnly,
+    public static BuildEnvironment ForDefaultFramework =>
```

Comment:
FYI - @radical.

The current code evaluates these Lazy properties in the static initializer. I refactored this so it is only touched when this property is touched.